tutorials/Tutorial11_Pipelines.ipynb

```diff
@@ -2,6 +2,12 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
```

Comment:
I'm sorry to ask you this, but as you can see, the diff for the tutorials is huge and it's quite hard for me to see where the changes are.

Hopefully there are not many changes though, so it would be really cool if you could do the following:
- Revert the notebooks as they are in `master` (even copy-pasting the raw version from GitHub is fine)
- Adding your changes *without running them*
- Commit the changes
- Run them, if you want, to make sure everything works well
  - If something is wrong, `git stash` and `git stash drop` the changes that PyCharm made while running the notebooks, modify them, commit again, and run again.
  
I understand it's quite a pain, so if you don't have the time, I'll just read through the diff anyway. Let me know! :slightly_smiling_face:

Reply:
Hi, I didn't run the notebooks, not sure where there are more changes than the exchange of "minimal" and "minimum". Maybe there is something related with the jump line characters used in Windows.

But just let me clarify what needs to be done, get the original notebooks from the master, make the changes (without running the notebooks), generate the markdown documents and send a new PR, is that correct?